spec/feature_tests/user_stories_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe 'user stories' do
+
+  let(:takeaway) { Takeaway.new(menu: dishes)}
+  let(:dishes)   { {starter: 4.95, main: 8.65, dessert: 5.35} }
+
+  # As a customer
+  # So that I can check if I want to order something
+  # I would like to see a list of dishes with prices
+  it 'so a customer can see a list of dishes, it has a menu including those dishes' do
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)